src/BeamAdapter/component/controller/AdaptiveBeamController.cpp

```diff
@@ -34,18 +34,16 @@
 #define SOFA_PLUGIN_BEAMADAPTER_ADAPTIVEBEAMCONTROLLER_CPP
 
 //////////////////////// Inclusion of headers...from wider to narrower/closer //////////////////////
+#include <BeamAdapter/config.h>
 #include <sofa/core/ObjectFactory.h>
 #include <sofa/defaulttype/VecTypes.h>
 #include <sofa/defaulttype/RigidTypes.h>
 
-#include <BeamAdapter/config.h>
 #include <BeamAdapter/component/controller/AdaptiveBeamController.inl>
 
-
-namespace sofa::component::controller::_adaptivebeamcontroller_
+namespace sofa::component::controller
```

Comment:
`AdaptiveBeamController` is in `_adaptivebeamcontroller_` so either you remove the `_adaptivebeamcontroller_` namespace in the header or you cancel this removal
(this is the reason macos is not compiling)